addons/website_slides/tests/test_embed_detection.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+from odoo import http
+
+
+class TestEmbedDetection(odoo.tests.HttpCase):
```

Comment:
Unfortunately yeah, it's not available in 12.0, the tests for that module are really small, there is not even a common.py for the tests of that module

Reply:
As we are about to drop the support for v12 (since v15 is coming), maybe we could reasonably consider only fixing this in v13 and later stables?
If that helps with tests and having "SlidesCase" available, I would say it's worth it.

Reply:
Well the fix is there already and the tests are working without the SlidesCase

That class doesn't really bring a lot of required stuff to the tests, mostly some slides, users, channels, etc
But it also inherit from common.SavepointCase, do you want me to make our test in 12.0 inherit from both HttpCase and SavepointCase ?

Reply:
I think Savepoint gets deprecated later in the stable versions but it might be required to make your test work actually since I see that "cls.env" is not available in setupClass? You may have to play a bit with that and find a working combination.

Reply:
for tracking: same than https://github.com/odoo/odoo/pull/74714

Reply:
Yes I need to inherit *something* to get the env in cls, I've tried savepointCase and transactionCase but neither are working, I've asked with my team as well we are trying to come up with a solution

Reply:
Well there is not a single instance of using setUpClass with HttpCase (I searched with regex HttpCase[\s\S]*.setUpClass\( and haven't found anything while searching for HttpCase[\s\S]*.setUp\( gave way more matches)
So I guess we'll have to go with setUp which works fine from what I tested